VirtoCommerce.CatalogModule.Web/Content/css/catalog.css

```diff
@@ -250,4 +250,13 @@
 }
 .capitalizeText {
     text-transform: capitalize;
-}
\ No newline at end of file
+}
+
+vc-item-search {
```

Comment:
I think there could be something more elegant. Probably you can discuss it with @anivirtoway.